### PR TITLE
Fix #119 delete default order by

### LINKED ISSUE #119: Supprimer les order_by par défaut et les ajouter là où il y a besoin
Dans les collections et dans l'export au moins.

Vérifier qu'il y a les tests pour le order_by là où on va l'utiliser, et adapter si besoin. Sinon ajouter.

### PR DESCRIPTION
**Pull request by flyusfly**

### CODE REVIEW

File: ban/tests/test_models.py [hunk 1]
```diff
@@ -537,3 +537,79 @@ def test_position_center_coerce(given, expected):
...
     else:
         assert not center
+
+
+def test_municipality_select_use_default_orderby():
+    mun1 = MunicipalityFactory(insee="90002")
+    mun2 = MunicipalityFactory(insee="90001")
+    # Check before data sort
+    assert models.Municipality.select().count() == 2
+    assert mun1.insee > mun2.insee
```
Comment (yohanboniface) [inline ban/tests/test_models.py]: You are testing python, doing so ;)

File: ban/core/models.py [hunk 2]
```diff
@@ -64,6 +60,9 @@ class Municipality(NamedModel):
     insee = db.CharField(max_length=5, unique=True)
     siren = db.CharField(max_length=9, unique=True, null=True)
 
+    class Meta:
+        order_by = ['insee']
```
Comment (yohanboniface) [inline ban/core/models.py]: I'd move them as class attribute of the Resource classes, and use it when building the Resource collection. The idea is not to have a default `order_by` after all ;)

File: ban/tests/test_models.py [hunk 3]
```diff
@@ -537,3 +537,79 @@ def test_position_center_coerce(given, expected):
         assert center.coords == expected
     else:
         assert not center
+
+
+def test_municipality_select_use_default_orderby():
```
Comment (yohanboniface) [inline ban/tests/test_models.py]: Those tests should be in the http modules, given we'll only use the order_by when building a collection for the API.

**Comment by flyusfly** [comment]
All is ready to merge...
Thx

File: ban/tests/http/test_municipality.py [hunk 4]
```diff
@@ -237,3 +237,14 @@ def test_delete_unknown_municipality_should_return_not_found(client, url):
     uri = url('municipality-resource', identifier=11)
     resp = client.delete(uri)
     assert resp.status == falcon.HTTP_404
+
+
+@authorize
+def test_municipality_select_use_default_orderby(get, url):
```
Comment (yohanboniface) [inline ban/tests/http/test_municipality.py]: Test name says "default order_by", but it's insee, right?

**Pull request merged**